solomonhawk/2022/src/dayn.rs.tmpl

```diff
@@ -12,7 +12,7 @@ pub fn part1(input: &[usize]) -> usize {
 }
 
 #[aoc(day3, part2)]
-pub fn part2(input: &[Vec<usize>]) -> usize {
```

Comment:
Nice that this file exists, I always just copy all the code from my previous day and delete the unnecessary bits, but this makes more sense.